packages/src/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Internal dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+const {
+    registerBlockType,
+    unstable__bootstrapServerSideBlockDefinitions, // eslint-disable-line camelcase
+} = wp.blocks;
+
+/**
+ * Function to register core blocks provided by the block editor.
+ */
+
+const registerGlutenblocks = () => {
```

Comment:
I'm not sure to understand how this method works. Are we really looping onto an empty array?

Reply:
> I'm not sure to understand how this method works. Are we really looping onto an empty array?

It is currently empty since there is no blocks in the package yet.

Reply:
I've added this comment `// Empty for now since there is no blocks in the package` in the empty array to avoid confusion